server/chat-plugins/mafia.ts

```diff
@@ -1514,7 +1516,11 @@ class MafiaTracker extends Rooms.RoomGame {
 		const targetString = self ? `You are` : `${user.id} is`;
 		if (!this.room.users[user.id]) return `${targetString} not in the room.`;
 		for (const id of [user.id, ...user.previousIDs]) {
-			if (this.playerTable[id] || this.played.includes(id)) return `${targetString} already in the game.`;
+			if (this.playerTable[id] || this.dead[id]) {
+				return `${targetString} already in the game.`;
```

Comment:
This function should probably be refactored to use `Chat.ErrorMessage` rather than returning a string on failure and seemingly false on success (?)

Reply:
(there's a few places in which mafia doesn't use new features like that, i/someone'll deal with that later)